ext/groonga/rb-grn-database.c

```diff
@@ -623,6 +624,36 @@ rb_grn_database_unmap (VALUE self)
     return Qnil;
 }
 
+/*
+ * Recreates all index columns.
+ *
+ * This method is useful when your index column is broken.
+ *
+ * @example
+ *   database.reindex
```

Comment:
It's an useless example. Example should show not only "how to use" but also "how to work". How about the following?

```ruby
database = Groonga::Database.create(:path => "/tmp/db")

Groonga::Schema.define do |schema|
  schema.create_table("Memos") do |table|
    table.short_text("title")
    table.text("content")
  end

  schema.create_table("Terms",
                      :type => :patricia_trie,
                      :key_type => :short_text,
                      :normalizer => "NormalizerAuto",
                      :default_tokenizer => "TokenBigram") do |table|
    table.index("Memos.title")
    table.index("Memos.content")
  end
end

database.reindex
# =>
#   Groonga["Terms.Memos_title"].reindex
#   Groonga["Terms.Memos_content"].reindex
```